tests/func-tests/monitoring_test.go

```diff
@@ -77,11 +77,11 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 			for _, rule := range group.Rules {
 				if rule.Alert != "" {
 					Expect(rule.Annotations).To(HaveKeyWithValue("summary", Not(BeEmpty())),
-						fmt.Sprintf("%s summary is missing or empty", rule.Alert))
+						fmt.Sprintf("%s summary annotation is missing or empty", rule.Alert))
```

Comment:
You don't need `fmt.Sprintf`. The `To` method can get format and parameters.
```suggestion
				Expect(rule.Annotations).To(HaveKeyWithValue("summary", Not(BeEmpty())), "%s summary annotation is missing or empty", rule.Alert)
```